### DISCUSSION TITLE
【福利】这是一张可以触摸互动的图片

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#1076] [其他]
![艾莲](https://github.com/user-attachments/assets/e550baa9-64af-41f8-9179-685e3468ed64)
# !一张可以与艾莲互动的*魔法图片*!👇
[![艾莲](https://github.com/user-attachments/assets/e550baa9-64af-41f8-9179-685e3468ed64)](https://www.bilibili.com/video/BV1GJ411x7h7/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "🤏")

**Comment by Badnuker** [comment]
我是防骗小助手，你猜我为什么在这

**Comment by Yui1026** [comment]
别信一楼，真的有互动

**Comment by hfdgcfgbcz** [comment]
那天这视频破亿了在座的各位都有责任(doge)

**Comment by LYBQ** [comment]
2楼说的对，互动是真的

**Comment by AnbulesPeo** [comment]
真的可以互动诶！

**Comment by su2569** [comment]
NGGYU-Rick

**Comment by ijog36** [comment]
好家伙真能互动
٩(˃̶͈̀௰˂̶͈́)و

**Comment by su2569** [comment]
治安局提醒您：建议安装国家反诈中心
![image](https://github.com/user-attachments/assets/169d1210-6cfd-44e4-a6d4-814d2b395d1d)
https://www.coolapk.com/apk/com.hicorenational.antifraud
https://apps.apple.com/cn/app/%E5%9B%BD%E5%AE%B6%E5%8F%8D%E8%AF%88%E4%B8%AD%E5%BF%83/id1552823102?l=en-GB
https://sj.qq.com/appdetail/com.hicorenational.antifraud?fromcase=40002&from_wxz=1

**Reply by fishqaq123** [reply]
竟然不是官方链接，一看就是假的！

**Comment by TBES-1** [comment]
兵不厌诈是吧

**Comment by user1-cloud** [comment]
额，看了大家的评论，对于我上面诈骗各位一事，我深感抱歉。所以，我决定补偿各位一个
# 触摸后能真正听到图中角色声音的好图！
[![知更鸟](https://github.com/user-attachments/assets/b152bacc-6992-4466-bf6c-e5ec9ffc642a)](https://www.bilibili.com/video/BV1qf42127cD/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "👆")

**Reply by Crafty-rabbit** [reply]
如果是假的，请去六分街录像店找我，我的名字叫“哲”

**Reply by 772099448** [reply]
是真的₍˄·͈༝·͈˄*₎◞ ̑̑

**Comment by Nainoz** [comment]
兵不厌诈

**Comment by Nina123973** [comment]
哇，居然是真的，真是太有意思辣

**Comment by lllliufeng** [comment]
;p愿者上钩

**Comment by QingMingqwq** [comment]
是真的诶!好厉害!

**Comment by figs23314** [comment]
每日上当1/1

**Comment by zifeiyu1111** [comment]
绳网也有电信诈骗啊？！

**Comment by shenguojia123** [comment]
是真的，我已经社保了**爽**

**Comment by huijin3** [comment]
欢愉信徒何时来？

**Comment by lllliufeng** [comment]
> [!CAUTION]
> 即将跳转到外部网站，谨防[诈骗](https://www.bilibili.com/video/BV1GJ411x7h7/)

**Comment by Rope-team** [comment]
绳网官方提醒您：建议安装新艾利都反诈中心

**Comment by LDR-NGM** [comment]
这互动，太劲了！
![R-C](https://github.com/user-attachments/assets/110d6773-a048-47f8-b4a4-b69d473cc3bd)

**Comment by AS111552** [comment]
奶奶滴，雀食没骗人

**Comment by OMO-114514** [comment]
byd气笑了

**Comment by bxzwq** [comment]
阿米诺斯

**Comment by Alice-warriors** [comment]
sensei！ wcnm
![Uploading img-257172464_1747987403652.jpg…]()

**Comment by owoshengjiang** [comment]
被骗力（悲）

**Comment by Aviaryd** [comment]
“真”互动

**Comment by kiana114** [comment]
你被骗了ദ്ദി˶>𖥦<)✧

**Comment by ltyuiop** [comment]
点击后一股强劲的音乐袭来🌚🌚

**Comment by kklk1880** [comment]
经典老梗😃

**Comment by mimikuku-39** [comment]
坏了，这是真的被骗了

**Comment by Robin-txdy** [comment]
每天上一当，当当不一样

**Comment by fengtian0307** [comment]
吃一堑，吃一堑（

**Comment by TheFreefiish** [comment]
年年考年年错
